### PR TITLE
bump versions

### PR DESCRIPTION
**Pull request by agent3bood**
Related issue https://github.com/gql-dart/gql/issues/287

### CODE REVIEW

**Review by micimize — APPROVED** [review]
seems fine

File: codegen/gql_build/lib/src/serializer_builder.dart [hunk 1]
```diff
@@ -69,12 +69,12 @@ class SerializerBuilder implements Builder {
     await for (final input in buildStep.findAssets(_generatedFiles)) {
       final lib = await buildStep.resolver.libraryFor(input);
       lib.units
-          .expand((cu) => cu.types)
+          .expand((cu) => cu.classes)
```
Comment (smkhalsa) [inline codegen/gql_build/lib/src/serializer_builder.dart:72]: Why is this change necessary?

Comment (agent3bood) [reply codegen/gql_build/lib/src/serializer_builder.dart:72]: The `types` is deprecated.

```
  /// Return a list containing all of the classes contained in this compilation
  /// unit.
  List<ClassElement> get classes;


  /// Return a list containing all of the classes contained in this compilation
  /// unit.
  @Deprecated('Use classes instead')
  List<ClassElement> get types;
```

**Review by smkhalsa — APPROVED** [review]

**Comment by agent3bood** [comment]
I am checking pub.dev and it looks somebody need to publish the new packages, currently it is published as alpha release.
